test/e2e/framework/framework.go

```diff
@@ -162,16 +162,7 @@ func New(kubeConfigPath string) (*Framework, CleanUpFunc, error) {
 		kubeConfigPath:            kubeConfigPath,
 		SchedulingClient:          schedulingClient,
 		OpenShiftMonitoringClient: osmclient,
-		ManifestsFactory: manifests.NewFactory(
-			namespaceName,
-			userWorkloadNamespaceName,
-			nil,
-			nil,
-			nil,
-			manifests.NewAssets("../../assets"),
-			&manifests.APIServerConfig{},
-			&configv1.Console{},
-		),
+		ManifestsFactory:          manifests.NewFactory(namespaceName, userWorkloadNamespaceName, nil, nil, nil, manifests.NewAssets("../../assets"), &manifests.APIServerConfig{}, &configv1.Console{}),
```

Comment:
Just for the future, I'd prefer we avoid reformatting changes like this. It just makes `git blame` more cumbersome to use.

Reply:
Ah, I used this new replace feature in Goland [to remove the last parameter from all `NewFactory` calls](https://github.com/openshift/cluster-monitoring-operator/pull/2078/commits/441c0465c9620b7cf99f6473610df90a4376ca71#diff-c354587e36151f73f4a4da62c22d612510bee633ada8a52ed16b89c7185f0283L174) and it ended up refactoring them too. Let me revert this, as this could lead to unnecessary merge conflicts down the line.